src/klaro.js

```diff
@@ -4,6 +4,14 @@ import 'scss/klaro.scss'
 // React Dev Tools browser extension.
 if(module.hot) require('preact/debug')
 
+import "core-js/modules/es.array.iterator";
+import "core-js/modules/es.array.from";
+import "core-js/modules/es.map";
+import "core-js/modules/es.set";
+import "core-js/modules/es.symbol";
+import "core-js/modules/es.symbol.iterator";
+import "core-js/modules/es.promise";
```

Comment:
If these are specific to IE11 support, can we add a comment to indicate why these lines got added.
Even better, we could link this PR or the issue.

Reply:
Would a `// This polyfills are needed for IE11 compatibility` work for you?

Reply:
Yes :)

(((*These)))